### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by aotarola**
- chore: change to a working ruby + bundler version
- docs: update metadata
- docs: update landing page links
- docs: remove sync driver references
- docs: remove demo link from the landing page
- docs: add a link to deprecated sync docs
- docs: use default `wd` driver
- docs: typofix

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -9,15 +9,14 @@ category: landing
...
 and even supports launching the app for you.
 
 Keep up to date with changes
-by checking the
-[releases](https://github.com/groupon/testium/releases).
+by checking the [releases][releases]
 
-<div style="text-align: center">
-  <iframe width="560" height="315" src="https://www.youtube.com/embed/qmPlM_SqZes" frameborder="0" allowfullscreen></iframe>
-</div>
```
Comment (aotarola) [inline index.md]: I figured that it should be removed until there's a new demo with modern code 🤔

File: _data/sidenav.yml [hunk 2]
```diff
@@ -10,29 +10,6 @@
...
-    - url: '/api/sync/#page'
-      title: Page
-    - url: '/api/sync/#alerts--confirms'
-      title: 'Alerts & Confirms'
-    - url: '/api/sync/#windows--frames'
-      title: 'Windows & Frames'
-    - url: '/api/sync/#pointer'
-      title: 'Pointer'
-    - url: '/api/sync/mixins.html'
-      title: 'Custom Methods'
```
Comment (dbushong) [inline _data/sidenav.yml]: Maybe just keep a single link to the old docs at the bottom of the nav with a "(deprecated)" note?

File: api/index.md [hunk 3]
```diff
@@ -103,10 +103,9 @@ Supported are `driver`, `keepCookies`, and `reuseSession`.
 #### `testium.browser`
 
 An instance of the requested driver implementation.
-The exact interface depends on the value of the [`driver`](/config.html#driver) setting:
+The exact interface depends on the value of the [`driver`](/config.html#driver) setting.
 
-* For "sync" see: [Synchronous Interface](/api/sync/)
-* For "wd" see: [Promise Chains](api/wd/)
+Currently, the recommended driver is "wd", see [Promise Chains](api/wd/)
```
Comment (dbushong) [inline api/index.md]: Not relevant for the docs (yet), but we should probably consider a major bump where we make `wd` the default

File: config.md [hunk 4]
```diff
@@ -34,20 +34,9 @@ and the grid will create a session for the best match it can find.
 
 Determines the interface for interacting with the browser.
 Testium will prepend the value with `testium-driver-` and treat the resulting string as an npm module name.
-The default value is "sync", so testium will try to load `testium-driver-sync`.
+The default value is "wd", so testium will try to load `testium-driver-wd` (the ["promise-chain" interface](/api/wd/) is powered by [`wd`](http://admc.io/wd/))
```
Comment (dbushong) [inline config.md]: it isn't yet :-(

File: setup.md [hunk 5]
```diff
@@ -34,10 +44,10 @@ driver = wd
 ```js
 const assert = require('assert');
 
-const browser = require('testium-mocha').browser;
+const { browser } = require('testium-mocha');
 
 describe('The homepage', () => {
-  before(browser.beforeHook());
+  before(browser.beforeHook({ driver: 'wd' }));
```
Comment (dbushong) [inline setup.md]: maybe prepend comment about how you don't need `{ driver: 'wd' }` if you have it in your `.testiumrc`

File: setup.md [hunk 6]
```diff
@@ -56,10 +66,10 @@ describe('The homepage', () => {
 You can just run the file above using `mocha`
 and testium will handle everything for you:
 
-* Launch the app and phantomjs.
+* Launch the app and a headless chrome.
```
Comment (dbushong) [inline setup.md]: remove "a"

File: api/index.md (hunk 3 above)
Comment (aotarola) [reply api/index.md]: Done!

File: config.md (hunk 4 above)
Comment (aotarola) [reply config.md]: It is [now](https://github.com/testiumjs/testium-core/releases/tag/v2.0.0)  🎉

**Pull request merged**